src/PolyType.SourceGenerator/SourceFormatter/SourceFormatter.FSharpUnion.cs

```diff
@@ -46,6 +46,7 @@ private static void FormatFSharpUnionCasesFactory(SourceWriter writer, FSharpUni
                     Type = {{unionCase.TypeModel.SourceIdentifier}},
                     Name = "{{unionCase.Name}}",
                     Tag = {{unionCase.Tag}},
+                    IsTagSpecified = true,
```

Comment:
The tag in F# unions just follows the order of union case declarations, so probably not something we can consider stable.
```suggestion
                    IsTagSpecified = false,
```